packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1066,6 +1067,7 @@ class _LinesState extends State<Lines> with AutoDisposeMixin {
           builder: (context, focusLine, _) {
             final isFocusedLine = focusLine == lineNum;
             return LineItem(
+              key: Key('Line Item $lineNum'),
```

Comment:
do we need to add keys just for testing? I think we can accomplish finding these widgets with test finders instead. For example you could look for all the LineItem widgets and grab the first instead of looking for Line Item 1

Reply:
Adding keys can have more side effects than just adding a debug tag to your widget, so in general, we should only use keys when we are using them for a reason rooted in the flutter lifecycle, not just for testing purposes. I always go back to this video, which gives a good explanation: https://www.youtube.com/watch?v=kn0EOS-ZiIc

Reply:
That was a helpful video! I've removed the keys. We want to make sure the gutter/lines get correctly scrolled into view (for go-to-line and for switching to a new stack frame), so when that happens we first visible line item won't be line item 1. 

I've switched to use finders to get the gutter item and line item based on their text content (eg '"40"` and `"count++"'`) and then verifying that they are horizontally aligned.